### PR TITLE
removing number from references

### PR DESCRIPTION
**Pull request by HassanAkbar**
Removing numbers from references as explained in https://github.com/metanorma/stepmod-utils/issues/194

### CODE REVIEW

File: spec/stepmod/converters/a_spec.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+require "spec_helper"
+require "support/smrl_converters_setup"
+
+RSpec.describe Stepmod::Utils::Converters::A do
+  let(:converter) { described_class.new }
+
+  context "format with number" do
+    describe "<a href=\"#eqn7\">(71)</a>" do
+      let(:xml_input) do
+        '<a href="#eqn7">(71)</a>'
```
Comment (ronaldtse) [inline spec/stepmod/converters/a_spec.rb:12]: Can you also help cover the case of `Figure <a ...>(1)</a>` Thanks!

**Review by ronaldtse — APPROVED** [review]

File: spec/stepmod/converters/a_spec.rb (hunk 1 above)
Comment (ronaldtse) [reply spec/stepmod/converters/a_spec.rb:24]: @HassanAkbar but we want to convert `Figure <a...>(1)</a>` into just `<<eqn1>>`?

**Comment by ronaldtse** [comment]
@HassanAkbar can you take care of this situation?

https://github.com/metanorma/stepmod-utils/pull/228#discussion_r1429790694

File: spec/stepmod/converters/a_spec.rb (hunk 1 above)
Comment (HassanAkbar) [reply spec/stepmod/converters/a_spec.rb:24]: @ronaldtse  currently I’m converting `<a href="#fig1">Figure (1)</a>` to `<<fig1>>` by adding the `href` inside the `<<>>` tag. Isn’t this the correct behavior? Do I need to change this?

Comment (ronaldtse) [reply spec/stepmod/converters/a_spec.rb:24]: I want to add one type of conversion in addition to what you have now:
```
Figure <a href="fig1">(1)</a>
```
=> 
```
<<fig1>>
```

Comment (HassanAkbar) [reply spec/stepmod/converters/a_spec.rb:24]: @ronaldtse So, from my understanding, we need to remove the prefix that appears before the `a` tag. Is this correct? If yes then which prefix do we need to remove? Only `Figure` and `Table` or are there some others?

Comment (ronaldtse) [reply spec/stepmod/converters/a_spec.rb:24]: Yes. Figure Table Formula. I think we need to check if there are other prefixes.

**Comment by HassanAkbar** [comment]
@ronaldtse The failing test case is because `expressir 1.3.0` is not available for `x86_64-linux` which is being fetched from the CI. 

Available expressir versions -> https://rubygems.org/gems/expressir/versions

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Please feel free to meege. Thanks!

**Comment by HassanAkbar** [comment]
@ronaldtse @opoudjis Version `0.4.9` was released with this fix.

**Pull request merged**